lib/URDNA2015Sync.js

```diff
@@ -238,6 +240,12 @@ module.exports = class URDNA2015Sync {
 
   // 4.8) Hash N-Degree Quads
   hashNDegreeQuads(id, issuer) {
+    if(this.deepIterations > this.maxDeepIterations) {
```

Comment:
I'm considering changing `deepIterations` to be tracked based on `id` -- to ensure that we're dealing with iterations per specific blank node vs. just having to call `hashNDegreeNQuads` once for a lot of nodes in a large graph (which should be fine).

Reply:
This has now been implemented.